agent/app/agent_capability.go

```diff
@@ -77,10 +77,13 @@ const (
 	capabilityExecCertsRelativePath                        = "certs"
 	capabilityExternal                                     = "external"
 	capabilityServiceConnect                               = "service-connect-v1"
+	capabilityGuardDuty                                    = "guard-duty-patrol"
 
 	// network capabilities, going forward, please append "network." prefix to any new networking capability we introduce
 	networkCapabilityPrefix      = "network."
 	capabilityContainerPortRange = networkCapabilityPrefix + "container-port-range"
+	//using guardDuty.service as a temporary placeholder
```

Comment:
nit: if this is not finalized, can you add a `TODO` here?

Reply:
Updated this